docs_src/microservices/application/GeneralAppServiceConfig.md

```diff
@@ -108,6 +108,28 @@ Only used for `messagebus` binding type
 | MessageBus SubscribeHost `Protocol` | tcp** | Indicates the protocol number for publishing to the Message Bus |
 | **MessageBus Optional** | ... | This section is used for optional configuration specific to the Message Bus type used. Please refer to [go-mod-messaging](https://github.com/edgexfoundry/go-mod-messaging/blob/master/README.md) for more details |
 
+### MqqtBroker
+
+This section defines the external MQTT Broker connect information.
+Only used for `external-mqtt` trigger binding type
+
+!!! note
+    `external-mqtt` is not the default Trigger type, so there are no default values for `MqttBrokerConfig` settings beyond those that the Go compiler gives to the empty struct. Some of those default values are not valid and must be specified, i.e. `Authmode`
+
+| Configuration                     | Default Value | Description                                                  |
+| --------------------------------- | ------------- | ------------------------------------------------------------ |
+| MqqtBroker `Url`            | blank**       | fully qualified URL to connect to the MQTT broker, i.e. `tcp://localhost:1883` |
+| MqqtBroker `ClientId`       | blank**       | ClientId to connect to the broker with                       |
+| MqqtBroker `ConnectTimeout` | blank**       | time duration indicating how long to wait before timing out                                                                            
```

Comment:
Casing is inconsistent for descriptions in this table.

Reply:
Cleaned up